model/model_updater.py

```diff
@@ -66,6 +81,20 @@ async def sync_model(self, hotkey: str) -> bool:
             )
             return False
 
+        if (
+            not isinstance(model.pt_model, GPT2LMHeadModel)
+            and not isinstance(model.pt_model, MistralForCausalLM)
+            and not isinstance(model.pt_model, LlamaForCausalLM)
```

Comment:
Can we put the allowlist into constants to make it easier to point to?

Reply:
Good point. Done.